### ISSUE TITLE
Error: need finite 'ylim' values

### ISSUE THREAD

**Issue by edwardphineas** [stephenturner/qqman#59]
So I have identified that my R version was causing some of my errors in trying to produce a plot from the dataset mentioned in a previous thread. 
I updated R and now I just get the single error:

Error in plot.window(...) : need finite 'ylim' values

Based on your comments on a previous post regarding this error, I checked the range of values on the y axis (i.e. the p value) using R's summary data:

       P          
 Min.   :0.00001  
 1st Qu.:0.24970  
 Median :0.49890  
 Mean   :0.49876  
 3rd Qu.:0.74200  
 Max.   :1.00000  
 NA's   :307     

So the p values are all >0 and <=1 as they should be.

As mentioned previously, if I try to omit NAs, I have 615363 of my 640669 rows omitted for reasons I do not yet understand.
In any case, whether I do this or not before the plot, I still get the error regarding finite ylim values.

Any help gratefully received.

datafile again for **reference:**

[mytestdata.txt](https://github.com/stephenturner/qqman/files/1354099/mytestdata.txt)

**Comment by sirselim** [comment]
I just happened to be creating a GWAS tutorial using this great package and thought I'd take a look at this issue. 

I'm not sure you're filtering out the SNPs with missing p-values correctly. I downloaded your data and have managed to generate a Mahattan plot without any issues after removing the missing data.

Here's the code I used:
```R
# load data as is
data.test <- read.table('mytestdata.txt', head = T, as.is = T)
# filter out missing p-values
data.sub <- data.test[!(is.na(data.test$P)),]
# generate Manhattan plot
qqman::manhattan(data.sub, chr = "CHR", bp = "BP", snp = "SNP", p = "P")
```

**Comment by stephenturner** [comment]
yes this was solved in #58
